torch/onnx/_internal/exporter/_core.py

```diff
@@ -552,6 +552,11 @@ def _handle_call_function_node_with_lowering(
     if _is_onnx_op(node.target):
         # Handle torch.ops.onnx.* ops. These ops can be directly added to the graph
         op_type, opset_version = _parse_onnx_op(node.target)  # type: ignore[arg-type]
+        # If final inputs are None, strip them from the node inputs
```

Comment:
Why do we need this?

Reply:
We have the logic in the builder that does this. Basically if an op has optional positional arguments it can carry Nones. In onnx trailing inputs being None is equivalent to not having those inputs. However for optimizers it is often easier if we normalize it (by removing the trailing inputs) to checking input counts etc.